### DISCUSSION TITLE
Sleep mode freezing with latest releases ~<0.110

### DISCUSSION THREAD

**Discussion by steverand** [seerge/g-helper#1086] [General]
When Ghelper checkbox "auto adjust windows power modes" is on, my laptop freezes (according to the logs) when it tries to go into Windows Modern Standby after being in sleep for some time. It took me weeks of Googling and experimenting with registry entries but as soon as I unchecked that it worked perfectly again. Powercfg output says my laptop does not support Windows Modern Standby.
Now with the latest update (0.110) it happens again but the option is no longer there. I had to roll back to 0.100. I think the latest update took the option away, so i'm assuming GHelper is still doing the Windows power modes automatically?

Anyway, no AC, only "asus optimization" service enabled, running 0.100 and no auto adjust power mode works fine for my A17FA706IC.

**Comment by seerge** [comment]
Answered your (same) question in your first topic https://github.com/seerge/g-helper/discussions/674#discussioncomment-6725468
